datadog/resource_datadog_logs_archive.go

```diff
@@ -43,7 +43,9 @@ func resourceDatadogLogsArchive() *schema.Resource {
 								Required:         true,
 								ValidateDiagFunc: validators.ValidateAWSAccountID,
 							},
-							"role_name": {Description: "Your AWS role name", Type: schema.TypeString, Required: true},
+							"role_name":       {Description: "Your AWS role name", Type: schema.TypeString, Required: true},
+							"encryption_type": {Description: "The type of encryption on your archive.", Type: schema.TypeString, Optional: true, Default: "NO_OVERRIDE"},
```

Comment:
```suggestion
							"encryption_type": {Description: "The type of encryption on your archive.", Type: schema.TypeString, Optional: true, Default: datadogV2.LOGSARCHIVEENCRYPTIONS3TYPE_NO_OVERRIDE, ValidateDiagFunc: validators.ValidateEnumValue(datadogV2.NewLogsArchiveEncryptionS3TypeFromValue)},
```

Reply:
Does sending a default value all the time without an encryption key change the behavior in any way?